thinkbox-deadline/azuredeploy.json

```diff
@@ -0,0 +1,388 @@
+                            }
+                        }
+                    }
+                ]
+            }
+        },
+
+
+        {
+            "apiVersion": "2015-05-01-preview",
+            "type": "Microsoft.Network/networkInterfaces",
+            "name": "[concat('nif', copyindex())]",
+            "location": "[parameters('location')]",
+            "copy":
+            {
+                "name": "nicLoop",
+                "count": "[parameters('numberOfSlaves')]"
+            },
+            "dependsOn": [
+                "Microsoft.Network/virtualNetworks/VNET",
+                "[concat('Microsoft.Network/publicIPAddresses/publicIp', copyindex())]"
+            ],
+            "properties": {
+                "ipConfigurations": [
+                    {
+                        "name": "ipconfig",
+                        "properties": {
+                            "privateIPAllocationMethod": "Dynamic",
+                            "subnet": {
+                                "id": "[variables('subnet-id')]"
+                            },
+                            "publicIPAddress":
+                            {
+                                "id": "[resourceId('Microsoft.Network/publicIPAddresses', concat('publicIp', copyindex()))]"
+                            }
+                        }
+                    }
+                ]
+            }
+        },
+
+        {
+            "apiVersion": "2015-05-01-preview",
+            "type": "Microsoft.Compute/virtualMachines",
+            "name": "azure-repo",
+            "location": "[parameters('location')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Storage/storageAccounts',parameters('storageAccount'))]",
+                "Microsoft.Network/networkInterfaces/nif",
+                "Microsoft.Compute/availabilitySets/availSet"
+            ],
+            "properties": {
+                "availabilitySet" :
+                {
+                    "id": "[resourceId('Microsoft.Compute/availabilitySets', 'availSet')]"
+                },
+                "hardwareProfile": {
+                    "vmSize": "Standard_A2"
+                },
+                "osProfile": {
+                    "computername": "azure-repo",
+                    "adminUsername": "[parameters('adminUserName')]",
+                    "adminPassword": "[parameters('adminPassword')]"
+                },
+                "storageProfile": {
+                    "osDisk": {
+                        "name": "osdisk",
+                        "osType": "Windows",
+                        "vhd": {
+                            "uri": "[concat('http://',parameters('storageAccount'),'.blob.core.windows.net/vhds/repo-osdisk.vhd')]"
+                        },
+                        "image":
+                        {
+                            "uri": "https://deadline.blob.core.windows.net/deadline72/azure-repo.vhd"

```

Comment:
The image is stored in a public container in the deadline storage account but the vhd should be stored in whatever storage account the user provides.